plugins/pack-deploy.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: pack-deploy
```

Comment:
I'm thinking if `grab-by-name` would be a better name?

Reply:
grab-deploy-by-name?

Reply:
The problem with deploy is that it sounds like deployment. Your plugin has a much broader scope than just deployments, so I would avoid to talk about that.

Reply:
@corneliusweig 
/me nods. Okay, my last try:  How about grab-yaml-by-name or pack-by-name? 
Sounds reasonable for you to move forward with the current approach or you want before the merge to use labels?